test/board_test.exs

```diff
@@ -161,6 +161,18 @@ defmodule BoardTest do
       assert Board.move_status(board, user_input) == :out_of_bounds
     end
 
+    test "indicated the space is already taken" do
```

Comment:
Not related to this test, but the test above that tests a 2x2 board. Do we need that test?

Reply:
I included a different sized board test in this test group as I wanted to show that the current boards bounds mattered here and the out of bounds check wasn't just based off of the board being default 3x3. Without this test, if the upper bounds of the `is_out_of_bounds?` function is replaced by a hard coded 9, no other test fails and it looks like that is an OK change if that makes sense, so although not necessary I think the 2x2 test gives the developer confidence that the out of bounds check works on whatever bounds the given board is, not a hard coded number. This all stemmed from a conversation Eli and I had about if you have a function where you hard code an input and no tests fail, you could be missing test coverage. What are your thoughts?